specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-07-01/managedClusters.json

```diff
@@ -4183,6 +4188,31 @@
       "title": "The max price (in US Dollars) you are willing to pay for spot instances. Possible values are any decimal value greater than zero or -1 which indicates default price to be up-to on-demand.",
       "description": "Possible values are any decimal value greater than zero or -1 which indicates the willingness to pay any on-demand price. For more details on spot pricing, see [spot VMs pricing](https://docs.microsoft.com/azure/virtual-machines/spot-vms#pricing)"
     },
+    "ScaleDownMode": {
+      "type": "string",
+      "enum": [
+        "Delete",
+        "Deallocate"
+      ],
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/azure/virtual-machines/states-billing"
```

Comment:
not sure if that's the best way to url but that's what I found 😄

Reply:
I usually see these included just as a markdown link in the description or title. In this case, something like what we do for the `SpotMaxPrice` in the description.

I'm not sure how `externalDocs` gets rendered in SDKs. In REST API documentation it seems to be ignored (so there's no link shown, which is unfortunate).